spec/support/examples_group/shared_examples_for_vm_or_template_is_available.rb

```diff
@@ -1,30 +1,51 @@
 shared_examples_for "Vm operation is available when not powered on" do
   it "when powered on" do
     vm.update_attributes(:raw_power_state => power_state_on)
-    expect(vm.is_available?(state)).to be_falsey
+    if vm.respond_to?("supports_#{state}?")
+      expect(vm.public_send("supports_#{state}?")).to be_falsey
+    else
+      expect(vm.is_available?(state)).to be_falsey
+    end
```

Comment:
@Fryguy  The shared examples are called for multiple states such as start/stop too. And the code still contains the old methods validate_start. validate_stop etc. This logic is essential to not break the other code. It will be eventually removed when all caller code is refactored to use SupportsFeatureMixin.